hack/init-environment.sh

```diff
@@ -213,6 +213,45 @@ function install_secrets_manager () {
 }
 
 
+#####
+## Set up Google's CloudSQL Proxy to communicate with the CloudSQL database.
+#####
+function install-cloudsql-proxy () {
+  local -r kubeconfig=$1 env_dir=$2
+
+  # Configure helm
+  local -ra helm=($(configure_helm ${kubeconfig} ${env_dir}))
+
+  # Read cloudsql configuration info from vault
+  local -r vault_location=secret/dsde/monster/dev/command-center/cloudsql/instance
```

Comment:
```suggestion
  local -r vault_location=secret/dsde/monster/${env}/command-center/cloudsql/instance
```
Once the arg is passed in